package.json

```diff
@@ -11,11 +11,6 @@
     "test:e2e": "qavajs run --config test-e2e/api.ts",
     "debug:e2e": "qavajs run --config test-e2e/api.ts --tags @debug"
   },
-  "husky": {
```

Comment:
why did you remove it?

Reply:
It does not work as is. I would make it work in combination with lint-staged, but @AlexGalichenko decided to get rid of hooks.

Reply:
got it